### ISSUE TITLE
Bristol, UK Parcels

### ISSUE THREAD

**Issue by justinelliotmeyers** [openaddresses/openaddresses#3157]
https://opendata.bristol.gov.uk/explore/dataset/asset-register-and-land-ownership/export/

@iandees or @nvkelso can one of you check out the license? I want to make sure our first UK source isokay

Thanks!

Addresses are in there!

**Comment by iandees** [comment]
It's [Open Government License](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/) and looks ok to me.

**Comment by nvkelso** [comment]
Agree it's `CC-BY`.

Doesn't look like 100% coverage, and some of the addresses are funky

<img width="1059" alt="screen shot 2017-06-23 at 11 07 23" src="https://user-images.githubusercontent.com/853051/27494793-3f105ab6-5804-11e7-89dc-1fffb741f940.png">
<img width="232" alt="screen shot 2017-06-23 at 11 07 12" src="https://user-images.githubusercontent.com/853051/27494794-3f13f41e-5804-11e7-936f-e3483f663775.png">
<img width="218" alt="screen shot 2017-06-23 at 11 07 03" src="https://user-images.githubusercontent.com/853051/27494795-3f15ce92-5804-11e7-8866-d9c4572b20e7.png">
<img width="232" alt="screen shot 2017-06-23 at 11 06 58" src="https://user-images.githubusercontent.com/853051/27494796-3f15b0ec-5804-11e7-8e9a-14cf4d16ffd6.png">

**Comment by justinelliotmeyers** [comment]
yes, names look goofy for some locations. In residential areas, names look good though (spot checked a few areas)
